src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/AudioPlayer/RecordAudio/RecordAudio.hooks.ts

```diff
@@ -0,0 +1,96 @@
+import { useState, useCallback } from 'react';
+
+export interface recorderControls {
```

Comment:
pls move this interface to RecordAudio.types.ts and if it's possible use type instead of interface to provide code consistency

Reply:
fixed